firefox_media_tests/playback/test_video_playback.py

```diff
@@ -48,10 +51,14 @@ def run_playback(self, timeout=0):
                     duration = min(timeout, duration_timeout)
 
                 try:
-                    # Some vendors don't start the videos at the beginning,
-                    # remembering where you were last time you watched.
-                    start_time = video.current_time
                     verbose_until(Wait(video, interval=1, timeout=duration),
                                   video, playback_done)
+                except TimeoutException:
```

Comment:
This is not a refactor, it's a fix. I remembered why we need to check the TimeoutException here. :)